cli/cmd/delete.go

```diff
@@ -32,7 +32,16 @@ var deleteCmd = &cobra.Command{
 	Short:        "Delete an existing stack",
 	Long:         "Delete the stack with the given name.",
 	SilenceUsage: true,
-	RunE:         runDelete,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Println("Running delete")
+		for _, stackName := range args {
+			err := runDelete(cmd, stackName)
+			if err != nil {
```

Comment:
Might make sense to message and continue to the next stack.

Reply:
Updated to log error and continue.